integration-tests/kubernetes/kubernetes.go

```diff
@@ -40,3 +40,18 @@ func CheckForPod(pods v1.PodList, targetRegex string) bool {
 
 	return false
 }
+
+// CheckPVC return true if number of pvcs in the list is equal to expected PVCs
+func CheckPVC(pvcs v1.PersistentVolumeClaimList, expectedPVC int) bool {
```

Comment:
Let this checks be within the test case itself

Reply:
done